### ISSUE TITLE
NVDA focuses on nothingness after any Windows notification

### ISSUE THREAD

**Issue by cyrmax** [nvaccess/nvda#14958]
### Steps to reproduce:
* Start reading some very long text without any keypresses (for example, large message in unigram);
* Receive any notification, for example from Chrome, Unigram, any other application;

### Actual behavior:
It is impossible to show the actual behavior with speech viewer.
The problem is that after few seconds after notification sound NVDA stops any ongoing speech and focuses on some pseudo-window. The only way to exit out of this window is to use alt+tab or any other keys that move system focus.
This problem happens only after updating to Windows 11.

### Expected behavior:
I expect that NVDA continues to read any ongoing text, after it reads notification text as usual and does not block interaction by focusing on some strange window.

### NVDA logs, crash dumps and other attachments:
Here is output from NVDA log viewer that shows information about this strange window.
```
INFO - globalCommands.script_navigatorObject_devInfo (13:45:25.572) - MainThread (13300):
Developer info for navigator object:
name: ''
role: Role.WINDOW
processID: 13824
roleText: None
states: exception: (-2147220991, 'An event was unable to invoke any of the subscribers', (None, None, None, 0, None))
isFocusable: exception: (-2147220991, 'An event was unable to invoke any of the subscribers', (None, None, None, 0, None))
hasFocus: False
Python object: <NVDAObjects.UIA.Toast_win10 object at 0x04E84A30>
Python class mro: (<class 'NVDAObjects.UIA.Toast_win10'>, <class 'NVDAObjects.behaviors.Notification'>, <class 'NVDAObjects.UIA.UIA'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <class 'garbageHandler.TrackedObject'>, <class 'object'>)
description: ''
location: None
value: exception: (-2147220991, 'An event was unable to invoke any of the subscribers', (None, None, None, 0, None))
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
appModule: AppModule(shellexperiencehost, appName='shellexperiencehost', processID=13824)
appModule.productName: 'Microsoft.Windows.ShellExperienceHost'
appModule.productVersion: '10.0.22621.1778'
appModule.helperLocalBindingHandle: c_long(182357448)
windowHandle: 66576
windowClassName: 'Windows.UI.Core.CoreWindow'
windowControlID: 0
windowStyle: -1811939328
extendedWindowStyle: 2097160
windowThreadID: 8000
windowText: ''
displayText: exception: cannot unpack non-iterable NoneType object
UIAElement: <POINTER(IUIAutomationElement) ptr=0xafc89e8 at 5184bc0>
UIA automationID: Exception: (-2147220991, 'An event was unable to invoke any of the subscribers', (None, None, None, 0, None))
UIA frameworkID: XAML
UIA runtimeID: (42, 66576, 4, 124)
UIA providerDescription: [pid:13824,providerId:0x0 Main(parent link):Unidentified Provider (unmanaged:Windows.UI.Xaml.dll)]
UIA className: Exception: (-2147220991, 'An event was unable to invoke any of the subscribers', (None, None, None, 0, None))
UIA patterns available: 
```

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Version: alpha-28326,62536a97 (2023.2.0.28326)

#### Windows version:
Windows 11 Education, version 22h2

#### Name and version of other software in use when reproducing the issue:
None

#### Other information about your system:
If this is important, I use high contrast Windows theme.

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Cannot reproduce on stable 2023.1

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
is this a duplicate of #14039?

**Comment by seanbudd** [comment]
Closing as duplicate https://github.com/nvaccess/nvda/issues/14039
